docs/how-to/cli-apps.rst

```diff
@@ -0,0 +1,157 @@
+=========================================
+Building your Console App with Briefcase
+=========================================
+
+Overview
+--------
+
+This guide explains the key differences when creating a **console application**
+using Briefcase, as opposed to a GUI-based app. It assumes you are already familiar
+with the basics of Briefcase and have completed the
+`BeeWare Tutorial <https://docs.beeware.org/en/latest/tutorial/tutorial-0.html>`_.
+
+We'll cover:
+
+1. How selecting "Console" changes your project configuration
+2. The ``console_app`` flag in ``pyproject.toml``
+3. How Briefcase handles command-line arguments
+4. Operational differences in how console apps run and display output
+5. Packaging differences specific to console apps (e.g., .pkg bundles on macOS)
+
+Creating a Console App
+-----------------------
+
+Start by generating a new Briefcase project with:
+
+.. code-block:: console
+
+    $ briefcase new
+
+When prompted to select a GUI framework, enter ``4`` to choose **Console**.
```

Comment:
This won't necessarily be option (4) - it might be a different value depending on the plugins that are installed.